src/export/bops/tests/applicationType.test.ts

```diff
@@ -0,0 +1,36 @@
+import { getBOPSParams } from "../index";
+import { DEFAULT_APPLICATION_TYPE } from "../model";
+
+describe("application_type is set correctly based on flowName", () => {
+  const testParams = {
+    breadcrumbs: {},
+    flow: { _root: { edges: [] } },
+    passport: {},
+    sessionId: "123"
+  };
+
+  it("defaults to `lawfulness_certificate` if we can't fetch current route", () => {
+    const generatedPayload = getBOPSParams({
+      ...testParams,
+      // @ts-ignore: Type 'undefined' is not assignable to type 'string'
+      flowName: undefined,
```

Comment:
The typechecker makes this test a bit redundant also we're not relying on fetching the flow name from the route anymore. Maybe we don't need this test?